web/src/app/browser/src/contextManager.ts

```diff
@@ -824,9 +824,9 @@ export default class ContextManager extends ContextManagerBase<BrowserConfigurat
 
     // Sets the default stub (as specified with the `getSavedKeyboard` call) as active.
     if(stub) {
-      return this.activateKeyboard(stub.id, stub.langId);
+      return this.activateKeyboard(stub.id, stub.langId, true);
```

Comment:
Without the added `true` parameter, the _first_ reload would remember the old keyboard, but the setting wouldn't be preserved for all subsequent reloads; the first-time default keyboard would appear instead.